server.js

```diff
@@ -17,37 +17,9 @@
 const express = require('express');
 const app = express();
 
-// Match e.g., "/hello", "/hello/foo", but not "/blah/foo.svg". Does not match "/": this is checked
-// in the conditional below on its own.
-const contentPageMatch = /^(\/\w+)+$/;
-
-// Disallow requests to "index.html" in any subdirectory (send a 302) along with
-// requests to a subdirectory that end with "/". This means that e.g., the
-// measure page, technically at "https://web.dev/measure/index.html", will
-// only be served under "https://web.dev/measure".
-// This resolves ambiguity for Service Workers and the Navaid router (which
-// internally trims trailing slashes for JS-based routing).
-const contentHandler = (req, res, next) => {
-  if (req.url === '/') {
-    req.url = '/index.html';
-  } else if (req.url.endsWith('/index.html') || req.url.endsWith('/')) {
-    const index = req.url.lastIndexOf('/');
-    const redir = req.url.substr(0, index);
-
-    // 301 means that Chrome and friends don't put the incorrect URL in the browser's history.
-    res.writeHead(301, {Location: redir});
-    return res.end();
-  } else if (contentPageMatch.exec(req.url)) {
-    req.url += '/index.html';
-  }
-
-  return next();
-};
-
 app.use(
-  express.static('dist', {index: false}),
-  contentHandler,
-  express.static('dist/en', {index: false, redirect: false}),
+  express.static('dist', {index: false, redirect: false}),
```

Comment:
pre-emptive comment: this folder should _never_ contain actual HTML content, so I figure disabling defaults here is fine

Reply:
Would you mind adding a comment to the code that explains that.
What's the benefit to disabling the defaults?

Reply:
I think your implicit comment is correct; I've reverted to the default. I think there are a bunch of edge cases here that I'm too worried about.